__integrations__/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@pagopa/io-functions-services-integrations",
+  "version": "1.0.0",
+  "description": "Integration test suite for integration testing",
+  "license": "MIT",
+  "keywords": [],
+  "author": "",
+  "scripts": {
+    "prestart": "npm-run-all generate-env",
+    "start": "docker-compose --env-file environments/env.base up -d",
+    "stop": "docker-compose --env-file environments/env.base down",
+    "stop:all": "docker-compose --env-file environments/env.base down && docker image prune -a -f && docker volume prune -f",
```

> docker image prune -a -f && docker volume prune -f

Not actually useful for local development (every successive run will need to fetch all layers from registry) nor for CI/CD (VMs are managed and I don't think we should care too much).

I suggest to remove it

```suggestion
    "stop:all": "docker-compose --env-file environments/env.base down",
```

---

This is already present as "yarn stop" command